python-sdk/src/astro/databricks/api_utils.py

```diff
@@ -18,6 +21,49 @@
 log = logging.getLogger(__name__)
 
 
+def delete_secret_scope(scope_name: str, api_client: ApiClient) -> None:
+    """
+    Delete the scope we created to prevent littering the databricks secret store with one-time scopes.
+
+    :param scope_name: name of scope to delete
+    :param api_client: populated databricks client
+    """
+    secrets = SecretApi(api_client)
+    try:
+        secrets.delete_scope(scope_name)
+    except HTTPError as h:
+        # We don't care if the resource doesn't exist since we're trying to delete it
+        if not h.response.json().get("error_code", "") == "RESOURCE_DOES_NOT_EXIST":
+            raise h
```

Comment:
It would be great to have a test covering this scenario. It would be nice as well if we didn't use single-characters to represent variables, `err` is probably more meaningful.